#! /bin/sh
patch -p1 -f $* < $0
exit $?

Enable sigjmp_buf.

--- newlib/newlib/libc/include/machine/setjmp.h	2008-02-29 23:46:46.000000000 +0100
+++ newlib/newlib/libc/include/machine/setjmp.h~	2008-02-29 23:41:19.000000000 +0100
@@ -251,7 +251,7 @@
 
 _END_STD_C
 
-#if defined(__CYGWIN__) || defined(__rtems__)
+#if 1 || defined(__CYGWIN__) || defined(__rtems__)
 #include <signal.h>
 
 #ifdef __cplusplus
@@ -259,7 +259,7 @@
 #endif
 
 /* POSIX sigsetjmp/siglongjmp macros */
-typedef int sigjmp_buf[_JBLEN+2];
+typedef int sigjmp_buf[_JBLEN+1+(sizeof (sigset_t)/sizeof (int))];
 
 #define _SAVEMASK	_JBLEN
 #define _SIGMASK	(_JBLEN+1)
