#! /bin/sh
patch -p1 -f $* < $0
exit $?

Bug fix.

--- newlib/newlib/libc/machine/i386/i386mach.h~	2000-08-28 19:50:06.000000000 +0200
+++ newlib/newlib/libc/machine/i386/i386mach.h	2008-02-21 14:55:52.000000000 +0100
@@ -25,7 +25,7 @@
  
 /* Use the right prefix for registers.  */
  
-#define REG(x) CONCAT1(__REG_PREFIX__, x)
+#define REG(x) __REG_PREFIX__ x
  
 #define eax REG(eax)
 #define ebx REG(ebx)
