#! /bin/sh
patch -p1 -f $* < $0
exit $?

Tweak the configuration.

--- binutils/ld/configure.tgt~	2008-08-09 07:35:13.000000000 +0200
+++ binutils/ld/configure.tgt	2008-11-18 15:39:17.000000000 +0100
@@ -201,6 +201,10 @@ x86_64-*-linux-*)	targ_emul=elf_x86_64
 			targ_extra_libpath=elf_i386
 			tdir_i386linux=`echo ${targ_alias}aout | sed -e 's/x86_64/i386/'`
 			tdir_elf_i386=`echo ${targ_alias} | sed -e 's/x86_64/i386/'` ;;
+x86_64-*-gnu*)		targ_emul=elf_x86_64
+			targ_extra_emuls="elf_i386"
+			targ_extra_libpath=elf_i386
+			tdir_elf_i386=`echo ${targ_alias} | sed -e 's/x86_64/i386/'` ;;
 i[3-7]86-*-sysv[45]*)	targ_emul=elf_i386 ;;
 i[3-7]86-*-solaris2*)	targ_emul=elf_i386_ldso
                         targ_extra_emuls="elf_i386 elf_x86_64"
